docs/zh_CN/_templates/index.html

```diff
@@ -7,7 +7,7 @@
 <div>
   <div class="chinese"><a href="https://nni.readthedocs.io/en/latest/">English</a></div>
```

Comment:
use zh link

Reply:
This link is for connecting English documents, so it should be `en`.